src/azure-cli/azure/cli/command_modules/backup/custom_help.py

```diff
@@ -305,7 +314,7 @@ def get_object_from_json(client, json_or_file, class_name):
     # Deserialize json to object
     param = client._deserialize(class_name, json_obj)  # pylint: disable=protected-access
     if param is None:
-        raise ValueError(
+        raise ResourceNotFoundError(
```

Comment:
Maybe `ValidationError` here. `ResourceNotFoundError` refers to 404 error for azure resources